packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2199,6 +2199,9 @@ export const components = ({
       },
       styleOverrides: {
         root: ({ theme }) => ({
+          position: "relative",
+          // to visually align input with label
+          insetBlockStart: `${2 / theme.typography.fontSize}rem`,
```

Comment:
Isn't `font-size` a `rem` value already?

Reply:
@KevinGhadyani-Okta From `theme` the `fontSize` value is a number `14`